doc/man3/EC_GROUP_copy.pod

```diff
@@ -181,9 +181,17 @@ EC_GROUP_method_of returns the EC_METHOD implementation in use for the given cur
 
 EC_GROUP_get0_generator returns the generator for the given curve or NULL on error.
 
-EC_GROUP_get_order, EC_GROUP_get_cofactor, EC_GROUP_get_curve_name, EC_GROUP_get_asn1_flag, EC_GROUP_get_point_conversion_form
-and EC_GROUP_get_degree return the order, cofactor, curve name (NID), ASN1 flag, point_conversion_form and degree for the
-specified curve respectively. If there is no curve name associated with a curve then EC_GROUP_get_curve_name will return 0.
+EC_GROUP_get_order() returns 0 if the order is not set (or set to zero) and 1 otherwise.
+
+EC_GROUP_get_cofactor() returns 0 if the cofactor is not set (or is set to zero) and 1 otherwise.
+
+EC_GROUP_get_curve_name() returns the curve name (NID) for the specified curve or will return 0 if no curve name is associated.
+
+EC_GROUP_get_asn1_flag() returns the ASN1 flag for the specified B<curve> .
+
+EC_GROUP_get_point_conversion_form() returns the point_conversion_form for the specified B<curve>.
+
+EC_GROUP_get_degree() returns the degree for the specified B<curve> or 0 if group_get_degree method is not populated.
```

Comment:
You could replace "`for the specified curve`" with "`for B<group>`", referring to the argument name in the `SYNOPSIS` entry.

Reply:
> or 0 if group_get_degree method is not populated

We should avoid to expose internal details here (because one day they might change).
We could either say a generic "or 0 on failure" or a little bit more detailed "or 0 if the operations is not supported by the underlying group implementation".

In this case, we should also evaluate the behaviour of existing implementations of `group_get_degree` inside the `crypto/ec/ecp*meth.c` files to check if there are other potential error values that should be documented (maybe there are not, I did not check this particuar function for now).